### ISSUE TITLE
Java docs generation error for janusgraph-driver during a build process.

### ISSUE THREAD

**Issue by porunov** [JanusGraph/janusgraph#1950]
Java docs error is received for `janusgraph-driver` during a build process. The folder `janusgraph-driver/target/apidocs` is empty after the build is successfully finished.

Expected behaviour:
Java docs are generated for `janusgraph-driver` module and populated in corresponding `apidocs` folder after the build is finished.

For confirmed bugs, please report:
- Version: 0.5.0-SNAPSHOT
- Steps to Reproduce: `mvn clean install -DskipTests=true`

### Stack Trace (if you have one)
```
...
[INFO] --- maven-javadoc-plugin:2.10.4:jar (attach-javadocs) @ janusgraph-core ---
[INFO] The goal 'org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:javadoc' has not been previously called for the module: 'org.janusgraph:janusgraph-driver:jar:0.5.0-SNAPSHOT'. Trying to invoke it...
[WARNING] Creating fake javadoc directory to prevent repeated invocations: /home/ws/janusgraph/janusgraph-driver/target/apidocs
[ERROR] Error fetching link: /home/ws/janusgraph/janusgraph-driver/target/apidocs/package-list. Ignored it.
[INFO] 
100 warnings
[WARNING] Javadoc Warnings
...
```

**Comment by porunov** [comment]
@farodin91 would you be able to check this issue please?

**Comment by porunov** [comment]
@farodin91 I've provided a fix in the PR #1951. Would be good if you could check it
